include/swift/AST/DiagnosticsSema.def

```diff
@@ -3310,6 +3310,9 @@ ERROR(trailing_closure_requires_parens,none,
       "trailing closure requires parentheses for disambiguation in this"
       " context", ())
 
+ERROR(opaque_type_var_no_init,none,
+      "property declares an opaque return type, but has no initializer "
```

Comment:
Do we want to call these 'initial value' or 'initializer expression' in diagnostics?